util/keyfile/src/lib.rs

```diff
@@ -3,63 +3,79 @@
 #![doc = include_str!("../README.md")]
 #![deny(missing_docs)]
 
+mod error;
 mod json_format;
-
+mod mnemonic_acct;
+pub use json_format::RootIdentityJson;
+pub use mnemonic_acct::UncheckedMnemonicAccount;
 pub mod config;
 pub mod keygen;
 
-use json_format::RootIdentityJson;
-use mc_account_keys::{PublicAddress, RootIdentity};
+use crate::error::Error;
+use bip39::Mnemonic;
+use mc_account_keys::{AccountKey, PublicAddress, RootIdentity};
 use mc_api::printable::PrintableWrapper;
-use std::{convert::TryInto, fs::File, io::prelude::*, path::Path};
+use std::{convert::TryInto, fs::File, io, io::prelude::*, path::Path};
```

Comment:
```suggestion
use std::{convert::TryInto, fs::File, io::Read, path::Path};
```

Reply:
Also needed io:write since we implement the trait. Made this change.